### ISSUE TITLE
Website submission page sometimes doesn't allow link submissions for audio

### ISSUE THREAD

**Issue by TrueAerosoul** [wilm0x42/wVote#79]
Unsure of cause at the moment. Causes both checkboxes be be selected at once (upload mp3 and link to soundcloud ex...)

**Comment by Tordek** [comment]
When picking the `link` option console reports `TypeError: Cannot read property 'startsWith' of null`, this regers to the line `						if (this.mp3Link.startsWith(host)) {`

and is probably caused by

```
					if (this.entry.mp3Format !== 'mp3') {
						// If it's a link, populate the field
						this.mp3Link = this.entry.mp3Url;
					}
```

**Comment by wilm0x42** [comment]
fixed in df963ac36e0a497bd7c1439b3c4067641a9db96c
